apps/kol-amelamdim/pages/register/index.tsx

```diff
@@ -69,6 +76,12 @@ const Register = () => {
             </Button>
           </Grid>
         </form>
+        <Typography component="h4">
+          כבר נרשמתם ?{' '}
+          <Link onClick={() => router.push('/register')}>
+            לחצו עלי להתחברות
+          </Link>
+        </Typography>
```

Comment:
Same here we should wrap the Typography with Link

Reply:
also, you can add sx={{mt: 2}} or something to make it spaced out from the button

Reply:
Resolved.